source/ranges.tex

```diff
@@ -3166,6 +3171,42 @@
 The expression \tcode{C | D} is well-formed if and only if
 the initializations of the state entities of \tcode{E} are all well-formed.
 
+\pnum
```

Comment:
I think the above "Given an additional range adaptor closure object" subclause probably need a `\pnum` too if this have one.

Reply:
Well, the "additional" here makes me believe it is tightly coupled with the preceding text, so there should be no `\pnum`.